service/webhooks_test.go

```diff
@@ -311,6 +311,71 @@ func (e *MockRancherClientFactory) GetClient(projectID string) (client.RancherCl
 	return *mockClient, nil
 }
 
+func TestTypeInListWebhooks(t *testing.T) {
```

Comment:
As it is, this test isn't really testing your code much bc in the mock implementation of List, you just filter out non webhook-service types.